google/cloud/bigquery/schema.py

```diff
@@ -556,3 +560,88 @@ def to_api_repr(self) -> dict:
         """
         answer = {"names": list(self.names)}
         return answer
+
+
+class SerDeInfo:
```

Comment:
Is this the name in the REST API? I'd prefer the full names rather than abbreviations, but I'm OK with this if it's to be consistent with the proto/REST API.

Reply:
`SerDeInfo` is the name in the proto/REST API definitions.